.gitignore

```diff
@@ -1,3 +1,5 @@
 intellij-java-google-style.xml
 **/target/
 **/.idea
+api-docs.json
+trading.iml
```

Comment:
https://www.notion.so/jarviscanada/Git-Github-GitFlow-Guidelines-4254192fe01e41bbbb2cce3f85185cda#4ea9e5dd98844e1286332e5c4eb5cec4

see this tip

Reply:
@jarviscanada Yes, I added it to my gitignore but it appears its still version controlled even after deleting the files and adding them again. I am going to do [this](https://gist.github.com/tsrivishnu/a2f3adbbca9fcad5f3597af301ad1abb)